packages/styling/src/styles/theme.ts

```diff
@@ -97,15 +97,17 @@ function _loadFonts(theme: ITheme): { [name: string]: string } {
   const lines = {};
 
   for (const fontName of Object.keys(theme.fonts)) {
-    const font = theme.fonts[fontName];
+    // tslint:disable-next-line:no-any
+    const font = (theme.fonts as any)[fontName];
     for (const propName of Object.keys(font)) {
       const name = fontName + propName.charAt(0).toUpperCase() + propName.slice(1);
       let value = font[propName];
       if (propName === 'fontSize' && typeof value === 'number') {
         // if it's a number, convert it to px by default like our theming system does
         value = value + 'px';
       }
-      lines[name] = value;
+      // tslint:disable-next-line:no-any
+      (lines as any)[name] = value;
```

Comment:
Fix the type of lines rather than adding a cast to any. Should be `const lines: { [name: string]: string } = {}`

Reply:
fixed